shared/dev/dumb-sheet.render.native.js

```diff
@@ -5,6 +5,9 @@ import dumbComponentMap from './dumb-component-map.native'
 import type {Props} from './dumb-sheet.render'
 import {Box, Text, Input, Button, NativeScrollView} from '../common-adapters/index.native'
 import {globalStyles} from '../styles'
+import {isTesting} from '../local-debug'
+
+const TEST_RENDER_NUMBER = 1
```

Comment:
i'll remove this. originally i was going to do like 5 at a time on a page and then switch but i moved away from that